README.md

```diff
@@ -379,7 +379,7 @@ You can convert structures using `Record()`, `Tuple()` (with the spread operator
 
 ```js
 const record = Record({ a: 1, b: 2, c: 3 });
-const record2 = Record.fromEntries([#["a", 1], #["b", 2], #["c", 3]]); // note that an iterable will also work
+const record2 = Record.fromEntries([["a", 1], ["b", 2], ["c", 3]]); // note that an iterable will also work
```

Comment:
While this is technically true, how many times have you seen `Object.fromEntries` where one entry was a string?

(btw, if you are on iOS there is a keyboard option to prevent it from turning `"` into `“` :stuck_out_tongue:)

Reply:
Updated the suggestion.

Reply:
It doesn't need to be an array-like, it only needs to be something where you can get a `"0"` and a `"1"` property. For example. Even `0` or `{}` are valid entries (and both yield to `#{ undefined: undefined }`).